src/app/home/cards/initial-settings/wallet-settings/wallet-settings.component.scss

```diff
@@ -0,0 +1,17 @@
+@import "src/styles.scss";
```

Comment:
Would be better to include colors in separate file, because styles.scss usually contains global selectors styling, and should be included only once, to not reapply those styles all over. The usual approach for colors is to create theme.scss, or colors.scss in assets/styles directory.